app/javascript/dashboard/components-next/dialog/Dialog.vue

```diff
@@ -48,6 +48,10 @@ defineProps({
     type: Boolean,
     default: false,
   },
+  maxWidth: {
+    type: String,
+    default: 'max-w-lg',
+  },
```

Comment:
This is not a great way to do this. I would suggest having more control over the prop, I tried something like that here.

https://github.com/chatwoot/chatwoot/blob/3a90604623b2f3be75a68c5719c26b1406d166ea/app/javascript/dashboard/components-next/dialog/Dialog.vue#L47-L51

https://github.com/chatwoot/chatwoot/blob/3a90604623b2f3be75a68c5719c26b1406d166ea/app/javascript/dashboard/components-next/dialog/Dialog.vue#L65-L76

This is from the ConversationFilter PR. You could do this.

In the previous case, the width prop accepts arbitrary values, without any validation. So I can add something like `max-w-lg hidden` and it will add it as it is. Or a typo like `mx-w-lg` will be accepted but not effect, since there is no validation

<details><summary> However I think there's a better solution. </summary>
<p>

You could remove the prop and related logic entirely, [disable the attribute](https://vuejs.org/guide/components/attrs#disabling-attribute-inheritance) inheritance completely and then attach the attributes to the `<dialog>` element using v-bind. ([ref](https://vuejs.org/guide/components/attrs#disabling-attribute-inheritance:~:text=We%20want%20all%20fallthrough%20attributes%20like%20class%20and%20v%2Don%20listeners%20to%20be%20applied%20to%20the%20inner%20%3Cbutton%3E%2C%20not%20the%20outer%20%3Cdiv%3E.%20We%20can%20achieve%20this%20with%20inheritAttrs%3A%20false%20and%20v%2Dbind%3D%22%24attrs%22%3A))

That way we can attach classes directly, and whenever we need a larger max-width, we can add it directly without a new prop.

</p>
</details> 

For now my suggestion is do the first implementation, later we can figure out the inherAttrs one.